scripts/train/train.py

```diff
@@ -545,14 +549,14 @@ def main(cfg: DictConfig) -> Trainer:
 
     eval_gauntlet_callback = None
 
-    if icl_tasks_config is not None:
+    if icl_tasks_config is not None and not use_async_eval:
```

Comment:
maybe make a jira to move the non ICL eval to this callback too after your other PR lands?

Reply:
Added eval loader in add7fbbee444f8434f5069a9defd00fcbb56b017 - actually a pretty small change